pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -608,8 +612,12 @@ func (gc *GarbageCollector) orphanDependents(owner objectReference, dependents [
 		go func(dependent *node) {
 			defer wg.Done()
 			// the dependent.identity.UID is used as precondition
-			patch := deleteOwnerRefStrategicMergePatch(dependent.identity.UID, owner.UID)
-			_, err := gc.patch(dependent, patch, func(n *node) ([]byte, error) {
+			p, err := patch.GenerateDeleteOwnerRefStrategicMergeBytes(dependent.identity.UID, []types.UID{owner.UID})
+			if err != nil && !errors.IsNotFound(err) {
```

Comment:
when does `NotFound` error occur?

Reply:
Oh, that was a copy paste error; fixed it now. thanks!